Assets/IPieceState.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```

Comment:
Ok, so if IPieceState was never defined we're going to need to define the interface correctly. Interfaces differ from classes in the following ways: